src/Init/WFTactics.lean

```diff
@@ -10,7 +10,7 @@ import Init.WF
 /-- Unfold definitions commonly used in well founded relation definitions.
 This is primarily intended for internal use in `decreasing_tactic`. -/
 macro "simp_wf" : tactic =>
-  `(tactic| simp [invImage, InvImage, Prod.lex, sizeOfWFRel,
+  `(tactic| try simp [invImage, InvImage, Prod.lex, sizeOfWFRel,
```

Comment:
I'd argue this one should be using `{ failIfUnchanged := false }`

Reply:
First, why?

Second, it's not possible. This is used very early on at a stage where the `config` argument is apparently discarded. With this change the build would fail in stage1 at Init/Data/Array/Basic.lean:663:6.

Reply:
Oh fun, it's a update-stage0 PR. It is possible, it's just annoying. The config argument isn't being discarded, it's being reinterpreted at the wrong type and probably the next boolean over has the actual value (this sometimes causes segfaults). I think you have to set `interpreter.prefer_native` to `true` in `stage0/src/stdlib_flags.h` to get it to build, and then run an update-stage0 and build lean again.

Reply:
> First, why?

It means that if `simp` fails in some other unpredictable way we don't just discard the error, Using `try` feels like the anti-pattern of Python's
```
try:
   v = d[k]
except:
   print("key was missing")
```
which actually catches all sorts of error that it would be best to propagate

> Second, it's not possible.

Ah, I had no idea that this touched stage0. Is `simp_wf` re-defined in a later stage anyway using the new syntax?

Reply:
Actually I think this PR doesn't need a update-stage0, you just need one between the previous PR and this one or else you can't use `failIfUnchanged` and expect `simp` to do something about it since it's still stage0 simp from before the previous PR.

Reply:
Okay, I will replace the `try` here after #2334 is merged.